pages/steps.tsx

```diff
@@ -22,13 +22,24 @@ const Steps: NextPage = () => {
   }
 
   return (
-    <div className="px-2 md:px-12 container md:mx-auto">
+    <div className="container px-2 md:px-12 md:mx-auto">
       <Stepper
         activeStep={activeStep}
         steps={steps.map((step) => step.label)}
       />
-      <div className={'mt-12'}>{currentComponent}</div>
-      <div className="mt-8 justify-start flex-wrap w-full flex justify-between md:justify-start">
+      <div className={'mt-12'}>
+        {(() => {
```

Comment:
One suggestion to make this look a litte bit cleaner:

Do not keep step configuration in store but declare it here in an external config object.
```
const config = {
  steps: [
    {
      name: 'userType',
      component: UserType,
    },
    // so on
  ]
}
```
The key here would be to have the component present in config as reference rather than string. This will allow us to get rid of that switch.

We already have the current step index in state, so the switch can be replaced by:
```
const StepComponent = config.steps[activeStep];

// Later in JSX

<div>
  <StepComponent />
</div>
```

Reply:
I'd also use a generic name instead of the translation key in the config. Using translation keys as identifiers feels a bit odd.